app/components/ThreadPost.jsx

```diff
@@ -173,24 +173,28 @@ module.exports = createReactClass({
   },
 
   renderThread: function() {
-    if (this.props.parent) {
+    if (this.props.isSearchResult) {
```

Comment:
if we indeed only want to show this particular action for search results, IMO it's better if we invert the logic and bail as soon as we find out that this isn't a `searchResult` instead of having to indent all of this code one level deeper. e.g.

```js
if (!this.props.isSearchResult) {
  return null;
}

[... rest of method]

```